### DISCUSSION TITLE
VB2 and CU2

### DISCUSSION THREAD

**Discussion by dcan999** [iNavFlight/inav#9679] [Ideas]
I am back with a much less intensive feature request than my last one 😂.

Now that we have access to all 6 ADC channels in INAV 7 (thanks devs), can we add an option on OSD to show VB2 and CU2? I think a decent amount of users would use a second battery voltage/current readout.

**Comment by dcan999** [comment]
Just wanted to bump this and ask if this will be supported in any upcoming releases like 8.0?

**Comment by sensei-hacker** [comment]
Custom OSD elements are on the way.
